File: app/src/main/java/com/example/test/mvvmsampleapp/view/adapter/CustomBindingAdapter.java
package com.example.test.mvvmsampleapp.view.adapter;

import android.databinding.BindingAdapter;
import android.view.View;

public class CustomBindingAdapter {
    @BindingAdapter("visibleGone")
    public static void showHide(View view, boolean show) {
        view.setVisibility(show ? View.VISIBLE : View.GONE);
    }
}

File: app/src/main/java/com/example/test/mvvmsampleapp/view/adapter/CustomBindingAdapter.kt
package com.example.test.mvvmsampleapp.view.adapter

import android.databinding.BindingAdapter
import android.view.View

@BindingAdapter("visibleGone")
fun showHide(view: View, show: Boolean) {
    view.visibility = if (show) View.VISIBLE else View.GONE
}